static/api/v1/openapi.yaml

```diff
@@ -5056,6 +5056,7 @@ components:
         - `client_secret_basic`: The client uses HTTP Basic as defined in
            OAuth 2.0, Section 2.3.1. Namely, `client_id` and `client_secret`
            are sent in the Basic Authorization header.
+        - `none`: The confidentiality is set to `public`.
```

Comment:
I believe all changes made to this file are version controlled in gitlab. But @BrendanSChang can speak to that.

Reply:
`"none"` is not a actual string value that can be supplied to represent a TokenEndpointAuthMethod. We're trying to say that:

> When token_endpoint_auth_method field is missing or set to `null`, it will be defaulted based on confidentiality:
> - public : none
> - confidential: client_secret_basic

Reply:
Apologies for the confusion, it is an actual value! Public clients don't authenticate with the token endpoint, which is what the string "none" represents.

Reply:
Got it, @tomsik68 can you make this change to the management API docs in the zero repo? And is there a timeline on which to release these changes/is this urgent?

Reply:
I don't think this is urgent, no one has asked for it yet.

Reply:
@BrendanSChang is this a PR we should merge or only make the change in zero repo?

Reply:
@turnipdabeets this MR should not be merged. These changes should only be made in the zero repo and updated here in PRs such as #259.